management/src/clusterctl/main.go

```diff
@@ -21,15 +21,32 @@ var (
 		},
 	}
 
-	extraVarsFlags = []cli.Flag{
+	extraVarsFlag = cli.StringFlag{
+		Name:  "extra-vars, e",
+		Value: "",
+		Usage: "extra vars for ansible configuration. This should be a quoted json string.",
+	}
+
+	cmdFlags = []cli.Flag{
+		extraVarsFlag,
+	}
+
+	commissionFlags = []cli.Flag{
+		extraVarsFlag,
 		cli.StringFlag{
-			Name:  "extra-vars, e",
+			Name:  "host-group, g",
 			Value: "",
-			Usage: "extra vars for ansible configuration. This should be a quoted json string.",
+			Usage: "host-group of the node(s). Possible values: service-master or service-worker",
 		},
 	}
 )
 
+// clusterctlFlags is used to save cmdFlags
+type clusterctlFlags struct {
```

Comment:
minor: `clusterctlFlags` can just be `flags` or `parsedFlags` as this is private structure inside clusterctl dir, so `clusterctl` seems a bit redundant.